tests/foreman/ui/test_branding.py

```diff
@@ -0,0 +1,57 @@
+"""Tests for branding
+
+:Requirement: Branding
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Branding
+
+:Assignee: akjha
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from robottelo.config import settings
+
+
+def test_check_version_login_screen(default_sat):
+    """Check correct version on the login screen
+
+    :id: f48110ad-29b4-49b1-972a-a70075a05732
+
+    :Steps: Curl the login screen
```

Comment:
```suggestion
    :Steps: Check Satellite version on login screen
```

Reply:
There is not much point in having one step in the docstring. How about:
Use curl to get the login screen
Use grep to get the version
Assert version in result.

If you think Line 35 is one step:
Use curl to get the login page and grep for the version.
Assert version in result.